app/res/drawable/logo_spacing.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list
+    xmlns:android="http://schemas.android.com/apk/res/android">
+    <item
+        android:drawable="@drawable/logo"
+        android:right="16dp"/>
```

Comment:
is this intentional to not have equal left and right values like the earlier `actionbar_modern_spacing` ?

Reply:
yess because as per figma its 16dp in right and by default it takes 16 in left

Reply:
left is predefined 16dp so no need to add that
Right is earlier 8dp but as per figma it should be 16dp